source/cfa.md

```diff
@@ -135,10 +135,11 @@ dimensions*, must be stored with the scalar aggregation variable's
 **`aggregated_dimensions`** attribute. The value of the
 **`aggregated_dimensions`** attribute is a blank separated list of the
 aggregated dimension names given in the order which matches the
-dimensions of the aggregated data. If the aggregated data is scalar
-then the value of the **`aggregated_dimensions`** attribute must be an
-empty string. The named aggregated dimensions must exist as dimensions
-in the netCDF dataset containing the aggregation variable.
+dimensions of the aggregated data, and following the CF group search
+algorithms. If the aggregated data is scalar then the value of the
```

Comment:
It might be good to point to the reference of the CF Conventions document, section 2.7 e.g. '[CF, see 2.7]' which outlines those algorithms, in line with Neil's question from https://github.com/NCAS-CMS/cfa-conventions/issues/40#issuecomment-1333555884 (I also wouldn't know where to look immediately to find these if I wanted to know more)?

Reply:
Hi Sadie - thanks for the comments. I've referenced CF as a whole in https://github.com/NCAS-CMS/cfa-conventions/pull/43/commits/3c190560f8283605e7010e7cb5b7a2d435b3bd00, but didn't mention the section since we don't know that it will always be that one. Is that OK?

Reply:
Sounds perfect (I wasn't aware the numbering could change, good point). Marking as resolved.